vtysh/vtysh.c

```diff
@@ -2448,6 +2448,22 @@ DEFUNSH(VTYSH_INTERFACE, vtysh_quit_interface, vtysh_quit_interface_cmd, "quit",
 	return vtysh_exit_interface(self, vty, argc, argv);
 }
 
+static void do_prepend(struct vty *vty, char *s, size_t n, const char *f, ...)
```

Comment:
So I'm not gonna block on this, but I have suggestion for making this even more DRY. There's an array containing the tokenized user input in each command handler function, `argv`. I dislike this setup of recreating the user's input. Instead we can do something like this (or exactly this):
```c
char *uinput = frrstr_join(argv, argc, " ");
snprintf(line, sizeof(line), "do %s\n", uinput);
XFREE(MTYPE_TMP, uinput);
```
Then just change this function sig to
```
do_prepend(struct vty *vty, char *line, size_t n, char **argv, char *argc)
```
No need for the `va_list` (these are hella expensive) in this function and callers dont have to rebuild the user input, their call becomes
```
do_prepend(vty, line, sizeof(line), argv, argc)
```
Thoughts?

Reply:
I did something different but based on your suggestion. Please, check.
I also pushed one more commit to fix the node for "show yang operational-data" command.